exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/AbstractElasticsearchExporterIntegrationTestCase.java

```diff
@@ -210,11 +211,13 @@ protected ElasticsearchExporterConfiguration getDefaultConfiguration() {
   public static class ElasticsearchTestClient extends ElasticsearchClient {
     private static final ObjectMapper MAPPER = new ObjectMapper();
 
-    protected final RecordIndexRouter indexRouter;
+    private final RecordIndexRouter indexRouter;
+    private final RestClient client;
```

Comment:
:thought_balloon: Both of these are temporary and will be removed at the end when this test case is deleted.